march_rqt_gait_selection/src/march_rqt_gait_selection/gait_selection_view.py

```diff
@@ -1,12 +1,11 @@
 from PyQt5.QtCore import Qt
-from PyQt5.QtWidgets import QWidget
+from PyQt5.QtWidgets import QComboBox, QLabel, QWidget
 from python_qt_binding import loadUi
-import rospy
 
 from .gait_selection_pop_up import PopUpWindow
 
 
-AMOUNT_OF_AVAILABLE_SUBGAITS = 6
+DEFAULT_AMOUNT_OF_AVAILABLE_SUBGAITS = 3
```

Comment:
Why do you need this? Can't you call the add subgait menus just everytime you need to make a menu with a certain amount of subgaits. Then every menu has the right amount.

Reply:
It does add subgait menu's and labels if necessary but for someone who has never seen the rqt gait selection (or barely) it would be nice to have a few defaults just to understand the GUI better. It would be possible to have 0 menu's as default but for the sake of clarity I've added just a few defaults.

Reply:
May be it is nicer to set the default to one, as there are also gaits with one subgait.